### ISSUE TITLE
Set default lambda node version to node12 for Cognito Lambda Triggers

### ISSUE THREAD

**Issue by hisham** [aws-amplify/amplify-cli#4357]
**Is your feature request related to a problem? Please describe.**
I'm using a library that requires node 12 features, but by default amplify cli creates lambdas with 				nodejs10.x runtimes. I keep forgetting to change them to nodejs12.x.

**Describe the solution you'd like**
Node 10 is old and node 12.x is currently the Active LTS. Amplify should use that as default, or make it configurable by us on a per project basis - what the default lambda runtime should be.

**Comment by kaustavghosh06** [comment]
@hisham All the functions generated using the `amplify add function` command is using node12 - in the latest version of the CLI at least. Is there any specific function that you're seeing which is using node10?

**Comment by hisham** [comment]
@kaustavghosh06 - the cognito lambda triggers which are added via amplify auth update. I was using cli 4.20.0 when generating them.

**Comment by kaustavghosh06** [comment]
I see - I believe it's the Cognito trigger templates which need to be updated specifically to node 12.

**Comment by michaelbrewer** [comment]
@hisham  i believe this already happened

**Comment by michaelbrewer** [comment]
otherwise i do have this: https://github.com/aws-amplify/amplify-cli/pull/7700

**Comment by erinmcgill** [comment]
I installed the amplify package the other day and when I did an `amplify add api`, all the CloudFormation templates still have `node10.x` as the runtime.

I've been able to fix all of them except for the `nested-cloudformation-stack.yml`. Can someone tell me where this comes from so I can update this as well?

I've tried downloading from my S3 bucket and reuploading with an updated runtime but it gets overwritten so my `amplify push` stacks keep failing and I am not sure how to get around it.
